<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fmProbaID</class>
 <widget class="QMainWindow" name="fmProbaID">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>203</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ввод пробы</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lePassword">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QToolButton" name="tbEnterPassword">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввод</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actTIns">
   <property name="text">
    <string>Вставить строку</string>
   </property>
  </action>
  <action name="actTDel">
   <property name="text">
    <string>Удалить строку</string>
   </property>
  </action>
  <action name="actPrIns">
   <property name="text">
    <string>Новый профиль</string>
   </property>
  </action>
  <action name="actPrDel">
   <property name="text">
    <string>Удалить профиль</string>
   </property>
  </action>
  <action name="actPrRen">
   <property name="text">
    <string>Переименовать профиль</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
